examples/basic/token/src/lib.rs

```diff
@@ -1,4 +1,4 @@
-#![cfg_attr(not(test), no_main, no_std)]
```

Comment:
As an option library user can add `#[no_std]` and have a `#[panic_handler]` in his contract.
For simplicity we can just opt out `#[no_std]` for examples.

Reply:
Also I've noticed zero difference in terms of wasm size of example with `#[no_std]` and without it.